contracts/pegged/tokens/MultiBridgeToken.sol

```diff
@@ -72,16 +72,18 @@ contract MultiBridgeToken is ERC20, Ownable {
     }
 
     /**
-     * @dev Burns tokens from an address. Decreases total amount minted by the calling bridge.
+     * @dev Burns tokens from an address, deducting from the caller's allowance. Decreases total amount minted if called
+     * by a bridge.
      * @param _from The address to burn tokens from.
      * @param _amount The amount to burn.
      */
     function _burnFrom(address _from, uint256 _amount) internal returns (bool) {
         Supply storage b = bridges[msg.sender];
-        require(b.cap > 0, "invalid caller");
-        require(b.total >= _amount, "exceeds bridge minted amount");
-        unchecked {
-            b.total -= _amount;
+        if (b.cap > 0 || b.total > 0) {
+            require(b.total >= _amount, "exceeds bridge minted amount");
```

Comment:
this makes a previous bridge (cap was not zero) still able to burn after its cap and total become zero, maybe not what we want given the current sgn protocol

Reply:
I think this can be handled by monitoring whether cap and total are both zero and ignoring the burn if that becomes the case.

Reply:
it works, but should we consider cases where the bridge user mistakenly uses the old bridge contract, with the expectation that the burnt token would be bridged?

Reply:
anyway, i think there are solutions to this:
- set cap to 1 instead of 0 for disabled bridge
- set min_burn of the token in disabled bridge to max_int

I'm fine with this PR now